docs/iris/src/developers_guide/documenting/whats_new_contributions.rst

```diff
@@ -26,98 +22,54 @@ which improved Iris in some way. As such, a single Iris Pull Request may
-to Iris, targeting an Iris user as the audience.
-
-A contribution is a feature summary by the code author, which avoids the
-release developer having to personally review the change in detail :
-It is not in itself the final documentation content,
-so it does not have to be perfect or complete in every respect.
-
-
-Adding contribution files
-=========================
+Each contribution will ideally be written as a single concise bullet point
+in a reStructuredText format with a trailing blank line.  For example::
 
-Each release must have a directory called ``contributions_<release number>``,
-which should be created following the release of the current version of Iris. Each
-release directory must be placed in ``docs/iris/src/whatsnew/``.
-Contributions to the what's new must be written in markdown and placed into this
-directory in text files. The filename for each item should be structured as follows:
+  * Fixed :issue:`9999`.  Lorem ipsum dolor sit amet, consectetur adipiscing 
+    elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+    <blank link>
 
-``<category>_<date>_<summary>.txt``
-
-Category
---------
-The category must be one of the following:
-
-*newfeature*
-  Features that are new or changed to add functionality.
-*bugfix*
-  A bugfix.
-*incompatiblechange*
-  A change that causes an incompatibility with prior versions of Iris.
-*deprecate*
-  Deprecations of functionality.
-*docchange*
-  Changes to documentation.
+Note that this example also cites the related issue, optionally you may also
+include the pull request using the notation ``:pull:`9999```.  Where possible
+do not exceed **column 80** and ensure that any subsequent lines
+of the same bullet point is aligned with the first.  
 
-Date
-----
+For inspiration that may include adding links to code please examine past
+what's :ref:`iris_whatsnew` entries.  
 
-The date must be a hyphen-separated date in the format of:
+.. note:: The reStructuredText syntax will be checked as part of building
+          the documentation.  Any warnings should be corrected.  
+          `travis-ci`_ will automaticall build the documention when
+          creating a pull request, however you can also manually 
+          :ref:`build <contributing.documentation.building>` the documentation.
 
- * a four digit year,
- * a three character month name, and
- * a two digit day.
+.. _travis-ci: https://travis-ci.org/github/SciTools/iris
 
-For example:
-
- * 2012-Jan-30
- * 2014-May-03
- * 2015-Feb-19
-
-Summary
--------
-
-The summary can be any remaining filename characters, and simply provides a
-short identifying description of the change.
-
-For example:
-
- * whats-new-aggregator
- * using_mo_pack
- * correction-to-bilinear-regrid
- * GRIB2_pdt11
+The content of the bullet point should highlight the change that has been made
+to Iris, targeting an Iris user as the audience.  Ideally, each bullet point
```

Comment:
@tkknight The last sentence is incomplete... did you need this or snip the rest of the content by mistake?

Reply:
cut/paste error - will fix.